include/libopencm3/sam/common/pmc_common_all.h

```diff
@@ -27,160 +34,223 @@
 
 /* --- Power Management Controller (PMC) registers ----------------------- */
 
-/* System Clock Enable Register */
+/** System Clock Enable Register */
 #define PMC_SCER			MMIO32(PMC_BASE + 0x0000)
```

Comment:
Things like these are not problematic.

Reply:
(Answered as of https://github.com/libopencm3/libopencm3/pull/1411#discussion_r917442391)